lms/views/exceptions.py

```diff
@@ -65,6 +65,11 @@ def hapi_error(self):
     )
     def validation_error(self):
         self.request.response.status_int = self.exception.status_int
+        LOG.info(
```

Comment:
ValidationError produces the full stacktrace of the exception which can
be a bit noisy.

Replace it with a more focus log line that includes the ValidationError
messages and the full list of request parameters which might be useful
in debugging.

The current full stack trace also exposes the full list of request
parameters.